avocado/core/job.py

```diff
@@ -549,7 +561,7 @@ def run_tests(self):
                        sys.argv)
 
         # This is "almost ready" for a loop
-        summary = self.test_suite.run(self)
+        summary = self.test_suites[0].run(self)
```

Comment:
Here is a good example of my previous point (https://github.com/avocado-framework/avocado/pull/4041/commits/2c84265255305ebadd5c121f5a4b659745ce08e4#r463876297).

Reply:
Yes, this is temporary. The idea is to insert a for loop here.